schema/Session.js

```diff
@@ -139,9 +139,8 @@ cube(`GroupedSessionRetention`, {
           SELECT
             session_id,
             plio_id,
-            user_id,
-            UNNEST(string_to_array(retention, ','))::int AS retention_array
-          FROM ${GroupedSession.sql()} AS session
+            user_id
+          FROM ${GroupedSession.sql()} AS session, UNNEST(SPLIT(session.retention, ',')) AS retention_array
```

Comment:
if we are making this change - then, we cannot say that we support RDS too - it has to be BQ only, right?

Reply:
yeah right. The query is still failing, I'm checking again.